free-1.md

```diff
@@ -1,43 +1,53 @@
-# Free project 1
+# Music Mixing
 
 ## The user and a language
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+This project is mostly targetted at people who have a general interest in audio mixing, but have neither the background nor the expensive software typically used (e.g. Ableton). Additionally, for the less serious music enthusiast, this DSL would be a significantly lighter weight and cheaper alternative to currently existing music software. This DSL would help users learn more about mixing music by providing them basic and easy to understand tools to manipualte sounds as well as allowing them to develop their own tools if necessary. For instance, some built-in features of the language might be basic volume adjustments at specified intervals or  allowing for a certain segment of a sound to be looped.
+
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+This DSL allows users to get their food in the door when it comes to mixing music. Given how popular electronic music has become, I imagine that several fans probably have a few of their own novel ideas, but no outlet for them. I know personally that most existing audio mixing software is not only very pricey, but also extremely complicated. Simply learning how to use the basics requires several hour long tutorials on Youtube. This DSL plans to help amateurs who simply want some exposure to audio mixing without the time or capital investments.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+I think a DSL would be appropriate because, if designed effectively, audio mixing can be made more user-friendly. Most audio mixing software emulates a DJ console (the thing people use with a lot of knobs and dials and what not). Although it may seem counterintuitive, I felt audo mixing can be more effectively done through a language rather than a visual gui of a DJ console. To me, all the knobs and dials are pretty intimidating, and make it difficult to express ideas.
 
```

Comment:
Great point! I think a lot of people with a good ear for music are filled with new ideas but lack the technical expertise to learn all of the confusing software. They also might be intimidated by all of the different technologies and then hold themselves back from starting to learn anything. This language could be extremely effective if it was marketed as something simple and easy to learn.